Microsoft.Toolkit.Uwp.UI.Controls.Lottie/LottieAnimationView.cs

```diff
@@ -0,0 +1,1108 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Graphics.Canvas;
+using Microsoft.Toolkit.Uwp.UI.Controls.Lottie.Model;
+using Microsoft.Toolkit.Uwp.UI.Controls.Lottie.Value;
+using Newtonsoft.Json;
+using Windows.Data.Json;
+using Windows.UI;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Automation.Peers;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls.Lottie
+{
+    /// <summary>
+    /// This view will load, deserialize, and display an After Effects animation exported with
+    /// bodymovin (https://github.com/bodymovin/bodymovin).
+    /// <para>
+    /// You may set the animation in one of two ways:
+    /// 1) Attrs: <seealso cref="FileNameProperty"/>
+    /// 2) Programatically: <seealso cref="SetAnimationAsync(string)"/>, <seealso cref="Composition"/>,
+    /// or <seealso cref="SetAnimationAsync(TextReader)"/>.
+    /// </para>
+    /// <para>
+    /// You can set a default cache strategy with <seealso cref="CacheStrategy.None"/>.
+    /// </para>
+    /// <para>
+    /// You can manually set the progress of the animation with <seealso cref="Progress"/> or
+    /// <seealso cref="Progress"/>
+    /// </para>
+    /// </summary>
+    public class LottieAnimationView : UserControl, IDisposable
```

Comment:
Make this a partial class to split our properties and such like we've done with other controls?

Reply:
Done.